app/tag-management/all-tags/tag-list/tag-list.component.css

```diff
@@ -0,0 +1,25 @@
+.tag-list {
+  list-style-type: none;
+}
+
+.tag-header {
+  display: flex;
+  flex-direction: row;
+  align-items: baseline;
+  justify-content: space-between;
+}
+
+.tag-header h3 {
```

Comment:
do not alter headlines. We want to be consistent on all our sub-sites, so headlines are defined in the overall css file `hip-theme`